hubble/hubble.go

```diff
@@ -539,6 +568,12 @@ func (k *K8sHubble) Enable(ctx context.Context) error {
 		warnFreePods = append(warnFreePods, podsName)
 	}
 
+	if peerSvc := k.generatePeerService(); peerSvc != nil {
```

Comment:
I'm fine with the changes but it's up to you now, does it matter if the peer is only created after Relay and hubble? If it matters then the services needs to be created on line 561, otherwise here is fine.

Reply:
No, it won't matter. Even if the relay fails to pick up the peer service (which it won't because it needs some time to spin up), it will try again later.

Reply:
alright let's merge it